.github/workflows/js-build-and-lint.yml

```diff
@@ -85,8 +92,7 @@ jobs:
           git push origin
 
       - name: tsc
-        if: steps.changes.outputs.frontend == 'true'
-        continue-on-error: true
```

Comment:
This is not what we want (it means if tsc fails, it moves on to next step without failing job).